src/libexec/zmfixperms

```diff
@@ -265,6 +265,18 @@ if [ -d /opt/zextras ]; then
     fi
   fi
 
+  if [ -d /opt/zextras/docs ]; then
+    printMsg "Fixing permissions and ownership on /opt/zextras/docs"
+    if [ -d /opt/zextras/docs/conf ]; then
+        docs_owner=$(stat -c %U /opt/zextras/docs/conf)
```

Comment:
Why do we need a stat? Isn't the user deterministic?

Reply:
deleted it